docs/BR.md

```diff
@@ -704,6 +704,8 @@ The CA SHALL confirm that prior to issuance, the CA has validated each Fully-Qua
 1. When the FQDN is not an Onion Domain Name, the CA SHALL validate the FQDN using at least one of the methods listed below; and
 2. When the FQDN is an Onion Domain Name, the CA SHALL validate the FQDN in accordance with Appendix B.
 
+All DNS queries conducted in the course of validation MUST be made from the CA to authoritative nameservers, i.e. without the use of recursive resolvers operated outside the CA's audit scope.
```

Comment:
Could we move this to 3.2.2 instead, and rephrase as:

"All DNS queries conducted in the course of validation in Section 3.2.2.4, Section 3.2.2.5 and Section 3.2.2.8 MUST be made from the CA to authoritative nameservers, i.e. without the use of recursive resolvers operated outside the CA's audit scope."

Reply:
Today, a CA may use an external DNS resolver for the purpose of checking CAA according to 3.2.2.8 as long as this is done by a DTP (and according to the requirements for using a DTP) . Do we really want to change this? I think this is out-of-scope for this ballot. See BRF 1.3.2 for details - we should restrict the ballot to 3.2.2.4 and 3.2.2.5.

Reply:
I understand and agree, then it makes sense to include CAA lookups in this ballot.